pkg/domain/stateDiffSensitiveDoguConfig.go

```diff
@@ -0,0 +1,44 @@
+package domain
```

Comment:
🚫 **[checkstyle]** <dupl> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>1-44 lines are duplicate of `pkg/domain/stateDiffDoguConfig.go:1-44`